<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPropDiaClass</class>
 <widget class="QDialog" name="VideoPropDiaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPropDia</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>VidoeProperty</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>VideoSize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="videoSize_comboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>VideoFormat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="videoFormat_comboBox">
          <item>
           <property name="text">
            <string>mjpeg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yuv</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>FrameRate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_pushButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancle_pushButton">
       <property name="text">
        <string>Cancle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
